storage/pkg/command/frontend.go

```diff
@@ -293,11 +293,12 @@ func frontendConfigFromStruct(c *cli.Context, cfg *config.Config, filesCfg map[s
 							},
 						},
 						"version": map[string]interface{}{
-							"edition": "reva",
-							"major":   10,
-							"minor":   0,
-							"micro":   11,
-							"string":  "10.0.11",
+							"product": "oCIS",
+							"edition": "Community",
+							"major":   1,
+							"minor":   16,
+							"micro":   0,
+							"string":  "1.16.0",
```

Comment:
we could use `version.String` from "github.com/owncloud/ocis/ocis-pkg/version".

But that will hold a 3 digit version on releases and a hash on non-release builds

Reply:
Good idea! Will change it here.

Reply:
Could you please check if the solution is ok @wkloucek ?

Reply:
Hm... already noticed that I was not on a recent go version... sorry for the dependency mess